src/sxapi/cli/cli_user.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+import keyring
+
+from sxapi.base import (
+    IntegrationAPIV2,
+    PublicAPIV2,
+)
+
+
+class CliUser:
+    """
+    CliUser class used for initializing, storing, retrieving and deleting
+    credentials and creating/holding Instances of supported API
+    Client.
+
+    This class should only be used in the cli package.
+    """
+
+    def __init__(self):
+        """
+        Basic User Credentials Constructor
+
+        calls self._init_creds() to set available credentials on startup.
+        """
+
+        self.token_env_names = "SMAXTEC_TOKEN"
```

Comment:
please specify what kind of token this is. 

we do have several token types in our system. Access token api, firestore tokens, long lived access tokens, etc 

we might also have different tokens for read/write or certain APIs.

`self.api_access_token` might suffice

Reply:
renamed to `self.api_access_token` :)